modules/network-observability-SRIOV-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-SR-IOV-config_{context}"]
+= Configuring the monitoring of SR-IOV interface traffic
+In order to collect traffic from a cluster with Single Root I/O Virtualization (SR-IOV) device, the `FlowCollector` `spec.agent.ebpf.privileged` specification value must be `true`. Then, the eBPF agent monitors other network namespaces in addition to the host network namespaces, which are monitored by default. When a pod with a virtual functions (VF) interface is created, a new network namespace is created. The VF interface is migrated from the host network namespace to the pod network namespace.
+
```

Comment:
that isn't always the case though when using special sriov policy config that can happen also cc'd @bn222 for his comments

Reply:
Ok...I updated this to say "With `SRIOVNetwork` policy `IPAM` configurations specified, the VF interface is migrated from the host network namespace to the pod network namespace." Does that help? Or could be clearer?